src/esp/bindings/CoreBindings.cpp

```diff
@@ -14,25 +14,177 @@ namespace esp {
+      .def("set_vec3", [](Configuration& self, const std::string& key,
+                          const Magnum::Vector3& val) { self.set(key, val); })
+      .def("set_rad", [](Configuration& self, const std::string& key,
+                         const Magnum::Rad& val) { self.set(key, val); })
+
+      .def(
+          "find_value_location", &Configuration::findValue,
+          R"(Returns a list of keys, in order, for the traversal of the nested subconfigurations in
+          this Configuration to get the requested key's value or subconfig.  Key is not found if list is empty.)",
+          "key"_a)
+      .def(
+          "has_value", &Configuration::hasValue,
+          R"(Returns whether or not this Configuration has the passed key. Doesn not check subconfigurations.)",
+          "key"_a)
+      .def(
+          "has_bool",
+          [](Configuration& self, const std::string& key) {
+            return self.checkMapForKeyAndType(key, ConfigStoredType::Boolean);
+          },
+          R"(Returns true if specified key references a boolean value in this configuration.)")
+      .def(
+          "has_int",
+          [](Configuration& self, const std::string& key) {
+            return self.checkMapForKeyAndType(key, ConfigStoredType::Integer);
+          },
+          R"(Returns true if specified key references a integer value in this configuration.)")
+      .def(
+          "has_string",
+          [](Configuration& self, const std::string& key) {
+            return self.checkMapForKeyAndType(key, ConfigStoredType::String);
+          },
+          R"(Returns true if specified key references a string value in this configuration.)")
+      .def(
+          "has_float",
+          [](Configuration& self, const std::string& key) {
+            return self.checkMapForKeyAndType(key, ConfigStoredType::Double);
+          },
+          R"(Returns true if specified key references a float value in this configuration.)")
+      .def(
+          "has_quat",
+          [](Configuration& self, const std::string& key) {
+            return self.checkMapForKeyAndType(key,
+                                              ConfigStoredType::MagnumQuat);
+          },
+          R"(Returns true if specified key references a Magnum::Quaternion value in this configuration.)")
+      .def(
+          "has_vec3",
+          [](Configuration& self, const std::string& key) {
+            return self.checkMapForKeyAndType(key,
+                                              ConfigStoredType::MagnumVec3);
+          },
+          R"(Returns true if specified key references a Magnum::Vector3 value in this configuration.)")
+      .def(
+          "has_rad",
+          [](Configuration& self, const std::string& key) {
+            return self.checkMapForKeyAndType(key, ConfigStoredType::MagnumRad);
+          },
+          R"(Returns true if specified key references a Magnum::Rad value in this configuration.)")
```

Comment:
Not sure about usefulness of the `has_*` APIs, now that you have everything stored in a single map there isn't a possibility of the same key being used for two different types and so this could be expressed simply as this and there's no need to have a dedicated overload for each and every type.

```py
foo.get_type('bar') == mn.Rad
```

(assuming the above suggestion with `get_type()` returning a Python type object, which makes it *a lot* shorter than `ConfigStoredType.MagnumRad`)